src/storage/txn/scheduler.rs

```diff
@@ -428,14 +395,14 @@ impl<E: Engine> Scheduler<E> {
         };
         if let ProcessResult::NextCommand { cmd } = pr {
             SCHED_STAGE_COUNTER_VEC
-                .with_label_values(&[ctx.tag, "next_cmd"])
+                .with_label_values(&[tag, "next_cmd"])
                 .inc();
-            self.schedule_command(cmd, cb);
+            self.schedule_command(cmd, entity.cb);
         } else {
-            execute_callback(cb, pr);
+            execute_callback(entity.cb, pr);
```

Comment:
Why not call the callback inside thread pool?

Reply:
How?